### DISCUSSION TITLE
Installed Icon not visible to open

### DISCUSSION THREAD

**Discussion by sshsurabhi** [pawelsalawa/sqlitestudio#3823] [Q&A]
### Details
Hi, I installed this SQLitestudio in my Ubuntu 18.04 system through command line using "InstallSQLiteStudio-3.2.1" command. It has been installed and opened. But after I close the main window, I could not see any icon that reflects that I installed that application. I tried to installed the second time. But, I couldn't and getting an error message that I've already installed it. 

I checked with the "apt list", But there also it was showing that the application was not installed in my system. 

![Screenshot from 2020-07-24 11-23-07](https://user-images.githubusercontent.com/40215817/88377844-32dc9600-cda0-11ea-8751-f46c45c5aada.png)
![Screenshot from 2020-07-24 11-22-49](https://user-images.githubusercontent.com/40215817/88377856-3839e080-cda0-11ea-9835-1f992b2f5868.png)
![Screenshot from 2020-07-24 11-26-26](https://user-images.githubusercontent.com/40215817/88378057-8d75f200-cda0-11ea-80d8-084f489bb7b3.png)

![Screenshot from 2020-07-24 11-22-14](https://user-images.githubusercontent.com/40215817/88377863-3a03a400-cda0-11ea-98c2-5e9a939ac701.png)
![Screenshot from 2020-07-24 11-24-51](https://user-images.githubusercontent.com/40215817/88377924-54d61880-cda0-11ea-81e3-254c2d867108.png)

### Steps to reproduce
I went to the location folder and checked it. There was a folder created and a web file was there to open it again manually. 

Problem is that the SQLitestudio Icon is not producing and every time I need to go to the installed folder to open the application. 

### Operating system
Ubuntu 18.04.4 LTS

### SQLiteStudio version
SQLiteStudio-3.2.1

Please give me a solution. Thanks in advance.

**Comment by pawelsalawa** [comment]
It's there. It's the file named "sqliteatudio".

**Comment by sshsurabhi** [comment]
Sorry, I think, You did not understand my query. Application installed in my system but, I could not access it as simple as through Application Icon. Every time I need to go to the folder where that application was installed. When I search it from terminal to uninstall, it returns that there is no such application to uninstall. 



What is the simplest way to install that application to get an Icon in my application folder or else at least a simplest way to uninstall it from my system

**Comment by pawelsalawa** [comment]
I guess that the installer didn't create these icons and actualy this installer is going to be removed from SQLiteStudio.
With next version there will be no installer, just portable (download & unzip) distribution.

Maybe some Linux enthusiasts will create packages for your distribution, that will contain such icon links. Unfortunately I don't have time and enough expertise to do it right for all Linux distros. I have a faight in Linux community to create packages for distros that they need.
